installing/disconnected_install/installing-mirroring-creating-registry.adoc

```diff
@@ -23,7 +23,7 @@ If you already have a container image registry, such as Red Hat Quay, you can sk
 +
 [IMPORTANT]
 ====
-These requirements are based on local testing results with only release images and Operator images. Storage requirements can vary based on your organization's needs. You might require more space, for example, when you mirror multiple z-streams. You can use standard Red Hat Quay functionality to remove unnecessary images and free up space.
+These requirements are based on local testing results with only release images and Operator images. Storage requirements can vary based on your organization's needs. You might require more space, for example, when you mirror multiple z-streams. You can use standard link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/red_hat_quay_api_guide/index#doc-wrapper[Red Hat Quay functionality] and the proper link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/red_hat_quay_api_guide/index#deletefulltag[API callout] to remove unnecessary images and free up space.
```

Comment:
By standard, this means both the UI and the API. OMR *is* Quay underneath, so whatever works for Quay, works for OMR as well. Maybe link the entire Quay documentation here, not just the API, because the user can connect to the UI using the URL provided by the installer and they can log on with the provided credentials.

Reply:
Maybe something along the lines of "For more information about Quay UI and API usage, click here". Or some such.